atlp-rwanda/e-commerce-furebo-32-fn
# Implement custom views for seller's items and collections.

What does this PR do?
This PR implements custom views for the seller's items and collections.

Describe your changes
Single Item View:

Added a detailed view for sellers to see all relevant information about a single item they are selling, including price, description, images, and other details.
Optimized the view for clarity and usability, ensuring easy access and management of individual items.
Collection View:

Added a comprehensive view of all items in the seller's collection, presented in a grid or list format.
Included filtering and sorting options to help sellers quickly find specific items or organize their collections.
Updated the seller's dashboard to include links to the new views.

Enhanced the UI/UX for a seamless experience when navigating between individual items and collections.

Integrated with the backend to fetch and display item and collection data accurately.

Ensured responsiveness of all views on different devices.

PR Type
What kind of change does this PR introduce?

 Bugfix
 Feature
 Refactoring (no functional changes, no API changes)
 Documentation content changes
 Tests
 Other
How should this be manually tested?
Navigate to the seller's dashboard.
Click on an individual item to view its detailed information.
Navigate to the seller's collection to see all items in a grid or list format.
Test filtering and sorting options within the collection view.
Verify the accuracy and completeness of the item and collection data.
What are the relevant pivotal tracker stories?
[Pivotal Tracker Link](https://www.pivotaltracker.com/story/show/187570686)
Screenshots (if appropriate)
N/A
